commons/tools/metrics.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Metrics
+shortdesc: Gather metrics on your applications
+tags:
+- apps
+---
+
+# Clever Cloud Metrics
+
+<div class="panel panel-warning">
+  <div class="panel-heading">
+    <h4>Warning:</h4>
+  </div>
+  <div class="panel-body">
+    Clever Cloud Metrics is still in beta.
+  </div>
+</div>
+
+In addition to logs, you can have access to metrics to know how your application
+behaves. By default, system metrics like CPU and RAM use are available, as well
+as application-level metrics when available (apache or nginx status for instance).
+
+You can also publish your own metrics if you want (via `statsd`).
+
+## Activate metrics
+
+To enable metrics on your application you need to add `ENABLE_METRICS=true` to
+its [environment variables](/doc/admin-console/environment-variables/).
+Once it's done, metrics will automatically be gathered and displayed in the console.
```

Comment:
I find this line a bit misleading. Could you add a "and redeploy your application" ?